.github/PULL_REQUEST_TEMPLATE

```diff
@@ -4,6 +4,7 @@
 * [ ] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
 * [ ] I have added/updated documentation for all user-facing changes
 * [ ] I have added/updated test cases
+* [ ] I have included the rebuilt [production build of the client](https://libertem.github.io/LiberTEM/contributing.html?#building-the-client)
```

Comment:
Possibly should mention that this is only needed if the GUI code actually changed - we can't rely on people actually reading the documentation link, sadly :grin: